build/test_update_ext_version.py

```diff
@@ -0,0 +1,97 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import json
+import pathlib
+import pytest
```

Comment:
```suggestion
import json
import pathlib

import pytest
```

Obviously not using your isort extension. 😉